### PR TITLE
Cluster definition for operations data mart

### PR DESCRIPTION
**Pull request by adembo**
This PR adds a cluster definition for the Operations Data Mart blueprint. The definition should most certainly be considered a work in progress as it hasn't received any performance testing at all.

### CODE REVIEW

File: cloud-common/src/main/resources/application.yml [hunk 1]
```diff
@@ -159,6 +159,12 @@ cb:
   blueprint:
     cm:
       defaults: >
+                CDP 1.2 - Data Engineering: Apache Spark, Apache Hive, Apache Oozie=cdp-data-engineering-702;
+                CDP 1.2 - Data Mart: Apache Impala, Hue=cdp-data-mart-702;
+                CDP 1.2 - Operations Data Mart: Apache Impala, Hue, Apache Kudu, Apache Spark=cdp-op-data-mart-702;
+                CDP 1.2 - Operational Database: Apache HBase=cdp-opdb-702;
+                CDP 1.2 - SDX Light Duty: Apache Hive Metastore, Apache Ranger, Apache Atlas=cdp-sdx-702;
```
Comment (keyki) [inline cloud-common/src/main/resources/application.yml]: Can you please rebase the PR. I've merged a change that adds SDX blueprints for CDP 1.2. I'm not sure if you have made also a change there or just a copy with a new name.

Comment (adembo) [reply cloud-common/src/main/resources/application.yml]: Done.

**Comment by keyki** [comment]
The “remove HDFS” change is blocked on OPSAPS-52947

**Comment by adembo** [comment]
> The “remove HDFS” change is blocked on OPSAPS-52947

I've updated this PR to just include the new cluster definition, so we're no longer blocked on that JIRA.

**Comment by keyki** [comment]
The filename should have 701 as suffix instead of 710.

**Comment by adembo** [comment]
> The filename should have 701 as suffix instead of 710.

You sure? Here are the other filenames:
```./core/src/main/resources/defaults/clustertemplates/aws/aws-op-datamart-710.json
./core/src/main/resources/defaults/clustertemplates/aws/aws-dataengineering-710.json
./core/src/main/resources/defaults/clustertemplates/aws/aws-opdb-710.json
./core/src/main/resources/defaults/clustertemplates/aws/aws-datamart-710.json
```

**Comment by keyki** [comment]
> > The filename should have 701 as suffix instead of 710.
> 
> You sure? Here are the other filenames:
> 
> ```json
> ./core/src/main/resources/defaults/clustertemplates/aws/aws-dataengineering-710.json
> ./core/src/main/resources/defaults/clustertemplates/aws/aws-opdb-710.json
> ./core/src/main/resources/defaults/clustertemplates/aws/aws-datamart-710.json
> ```

Hm, okay let' leave it like that. I'll fix them next week. They all messed up. :)

**Pull request merged**